src/ui/screens/AddBountyScreen.tsx

```diff
@@ -39,22 +39,22 @@ export function AddBountyScreen({navigation}: {navigation: NavigationProp<AppSta
 
   const {calculatedBountyBond, bountyAllocationBN, isBountyValid, isBountyTitleValid} = useMemo(() => {
     if (bounty && bounty.bountyDepositBase) {
-      const calculatedBountyBond = calculateBountyBond(
+      const bountyBondCalculated = calculateBountyBond(
```

Comment:
Are the changes in this file part of this PR?

Reply:
No, but I was getting lint warnings, which were supposed to be fixed in adding bounty screen.